src/Workspaces/Core/Portable/FindSymbols/SymbolFinder.cs

```diff
@@ -15,7 +15,11 @@ namespace Microsoft.CodeAnalysis.FindSymbols
     public static partial class SymbolFinder
     {
         /// <summary>
-        /// Finds the symbol that is associated with a position in the text of a document.
+        /// Finds the symbol that is associated with a position in the text of a document. If 
+        /// multiple symbols are associated with the token (e.g. implicitly named anonymous type
```

Comment:
So, thinking about this, I think we really need to expose some API so people can handle this properly.  It seems somehow wrong that we're exposing something called "FindSymbolAtPosition" when the language is saying "but I can have multiple symbols at that position".  Can we make this into FindSymbolsAtPosition instead? (or, if that's a breaking change, can we have an overload to do that, and then just make this one call that one and do like ".FirstOrDefault"?  We can then audit our code and update things to call the new method and deal with it means if they get multiple symbols.